### ISSUE TITLE
check_updates problem with embedded perl

### ISSUE THREAD

**Issue by matteocorti** [matteocorti/nagios_plugins#27]
**Original reporter**: *mark.greenhalgh@hf-uk.com*

If I run the check_update on the cli it works fine but if I try and get nagios to run it then it comes up with this error.

-*ePN failed to compile /usr/lib64/nagios/plugins/check_updates: "Missing right curly or square bracket at (eval 2) line 395, at end of line 

I had to manually place the check_update file in this location as using the INSTALLSCRIPT option when compiling installed it in the wrong place.

**Comment by matteocorti** [comment]
**Original reporter**: *[matteo@corti.li](https://github.com/matteo@corti.li)*

Hi,

thanks for the report. The error means that there is a problem running the plugin with embedded perl. As a workaround you can start the plugin (in checkcommands.cfg)

with

```
/usr/bin/perl /usr/lib64/nagios/plugins/check_updates
```

instead of

```
/usr/lib64/nagios/plugins/check_updates
```

directly.

I will fix the problem (as the INSTALLSCRIPT one) as soon as possible.

Matteo

**Comment by matteocorti** [comment]
**Original reporter**: *[matteo@corti.li](https://github.com/matteo@corti.li)*

Fixed in r1019 (Version 1.1.0)
